mix.exs

```diff
@@ -57,4 +59,13 @@ defmodule ConfigCat.MixProject do
       {:ex_doc, "~> 0.23", only: :dev, runtime: false}
     ]
   end
+
+  defp package do
+    [ licenses: ["MIT"],
+      links: %{ "GitHub"=>"https://github.com/configcat/elixir-sdk" } ]
+  end
+
+  defp description do
+    "ConfigCat SDK for Elixir provides easy integration for your application to ConfigCat."
```

Comment:
Could we use this text instead:

`ConfigCat SDK for Elixir. Feature Flags created by developers for developers with ❤️. ConfigCat lets you manage feature flags across frontend, backend, mobile, and desktop apps without (re)deploying code. % rollouts, user targeting, segmentation. Feature toggle SDKs for all main languages. Alternative to LaunchDarkly. Host yourself, or use the hosted management app at https://configcat.com.`

Are there any limitations?

Reply:
Done. Not sure how it's going to look but we can fix it later if necessary 👍